graphql/util.go

```diff
@@ -764,6 +752,31 @@ func ModifyVersionHandler(ctx context.Context, dataConnector data.Connector, pat
 	if err != nil {
 		return mapHTTPStatusToGqlError(ctx, httpStatus, err)
 	}
+
+	if version.Requester == evergreen.PatchVersionRequester || version.Requester == evergreen.GithubPRRequester {
```

Comment:
can we use IsPatchRequester instead? (unless we explicitly don't want to include commit queue, although then we should explain that in a comment)

Reply:
done